api/src/service/cache2.spec.ts

```diff
@@ -509,8 +515,401 @@ describe("The cache", () => {
       assert.instanceOf(Result.unwrap_err(responseFromCache), NotFound);
     });
   });
+
+  describe("The BusinessEvent parsing", () => {
```

Comment:
please add a comment somewhere here or at the top of the file that describes the general setup in a few words. if you read top-down, you really wonder where the assertions are and what this is testing in the first place.